src/Workspaces/Core/MSBuild/MSBuild/BuildHostProcessManager.cs

```diff
@@ -53,11 +59,11 @@ public async Task<(IBuildHost, BuildHostProcessKind? PreferredKind)> GetBuildHos
         // us to discover VS instances in .NET Framework hosts right now.
         if (neededBuildHostKind == BuildHostProcessKind.NetFramework)
         {
-            if (!await buildHost.HasUsableMSBuildAsync(projectFilePath, cancellationToken))
+            if (!await buildHost.HasUsableMSBuildAsync(projectFilePath, cancellationToken).ConfigureAwait(false))
```

Comment:
how were tehse misssed before?  analyzer bug?

Reply:
This code previously lived in the LSP server project. Since that's not a library but instead the executable where we know we don't have UI threads, we have the ConfigureAwait() analyzer turned off. Once the file moved to the library, then it was required (and indeed, the tooling caught it all automatically.)